internal/feature/flags_test.go

```diff
@@ -6,54 +6,45 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestIsEnabledReturnsAnErrorIfKeyDoesntExist(t *testing.T) {
-	m := newStaticMapFeature(map[string]bool{})
-	assert.Panics(t, func() {
-		m.IsEnabled("foo")
-	})
+func TestGetReturnsFalseIfKeyDoesntExist(t *testing.T) {
```

Comment:
Do we want this to silently return false in our code? If we typo a feature checkflag name?

Reply:
We shouldn't be typo'ing checkflag names because we shoudl use feature.Foo instead of "foo". Maybe instead of map[string]Value it should be map[*struct{}]Value so that you *have* to use the right key?

Added the panic back. WDYT?

Reply:
Ahh I see, yeah this LGTM. Thanks!